runtime-sdk/src/modules/consensus_accounts/types.rs

```diff
@@ -1,4 +1,6 @@
 //! Consensus module types.
+use oasis_core_runtime::consensus::roothash::MessageEvent;
```

Comment:
Maybe nicer to import the re-exported type in `crate::types::message::MessageEvent`

Reply:
Indeed, fixed!